activemq/tests/test_check.py

```diff
@@ -12,5 +15,9 @@ def test(dd_agent_check):
     instance = {}
     aggregator = dd_agent_check(instance)
 
-    for metric in ACTIVEMQ_E2E_METRICS:
+    for metric in ACTIVEMQ_E2E_METRICS + JVM_E2E_METRICS:
         aggregator.assert_metric(metric)
+
+    aggregator.assert_all_metrics_covered()
+    # TODO: At the moment, JMX reported metrics are NOT in-app metrics, hence we can't assert the type yet.
```

Comment:
What do you mean here? That JMX metrics don't have metadata?

I'm fine with that PR overall but before hiding the problem, can we take the time to update and sync metadata for those jvm metrics? Metadata for it are stored in the core repo

Reply:
@FlorianVeaux 
About the `type` of the metric. The jmx metrics reported and stored in AggregatorStub are not in-app type, hence cannot be asserted correctly for the moment.